dbt_coves/utils/jinja.py

```diff
@@ -1,13 +1,25 @@
+import copy
+import os
+
 from jinja2 import ChoiceLoader, Environment, FileSystemLoader, PackageLoader
 
 
+def add_env_vars(context):
+    context_with_env_vars = copy.deepcopy(context)
```

Comment:
No need to deepcopy context, we can just modify it, I'd simplify this function by doing this:
```python
context['env_vars'] = os.environ.copy()
return context
```